src/components/Tooltip/Tooltip.tsx

```diff
@@ -115,6 +115,11 @@ interface TooltipBaseProps extends VibeComponentProps {
    * Overwrites z-index of the tooltip
    */
   zIndex?: number;
+  // TODO: make defualt next major
+  /**
+   * Limit tooltip to 240px
+   */
+  maxWidth?: boolean;
```

Comment:
Couldn't find a better name that will be short enough, open to suggestions

Reply:
setOfficialMaxWidth

Reply:
Even `withMaxWidth` or something similar, `limitWidth`